addons/payment_sips/controllers/main.py

```diff
@@ -36,15 +37,15 @@ def sips_ipn(self, **post):
             # meaningful information; log as a warning instead of a traceback
             _logger.warning('Sips: received empty notification; skip.')
         else:
-            self.sips_validate_data(**post)
+            self._sips_validate_data(**post)
         return ''
 
     @http.route('/payment/sips/dpn', type='http', auth="public", methods=['POST'], csrf=False)
     def sips_dpn(self, **post):
         """ Sips DPN """
         try:
             _logger.info('Beginning Sips DPN form_feedback with post data %s', pprint.pformat(post))  # debug
```

Comment:
```suggestion
            _logger.info('Beginning Sips DPN _handle_feedback_data with post data %s', pprint.pformat(post))
```
useless comment is useless + `form_feedback` no longer exists -> same for `sips_ipn`